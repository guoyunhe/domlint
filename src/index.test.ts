import * as members from '.';

test('members should match snapshots', () => {
  expect(members).toMatchSnapshot();
});
