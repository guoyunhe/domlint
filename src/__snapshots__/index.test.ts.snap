// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`members should match snapshots 1`] = `
{
  "DOMLint": [Function],
}
`;
